### DISCUSSION TITLE
Current AI task icon for flockdrones on them

### DISCUSSION THREAD

**Discussion by FlameArrow57** [stonepillars/goon-flock#203] [Ideas]
Wanted?

Would be similar to health icon, where Flockdrones have an ai task icon hovering by them, where you could see the Flockdrone's current ai task, or a different icon for if the Flockdrone is currently being controlled by another drone.

Think it would be nice for transparency, and an option could always be added to turn them off. Sometimes a drone's current task is a little unclear.

Uncertain if it's actually needed though.

**Comment by amylizzle** [comment]
God yes, that would be great. Probably make it toggleable though, because it might be a bit too cluttery.

**Comment by TobleroneSwordfish** [comment]
Note that you can examine a drone to see its task and we don't currently have icons for all tasks (the tgui ones won't work as overlays)
